services/web/server/src/simcore_service_webserver/statics_settings.py

```diff
@@ -48,3 +48,16 @@ def to_statics(self) -> Dict:
             exclude_none=True,
             by_alias=True,
         )
+
+
+class StaticsSettings(BaseSettings):
+    """used by the backend"""
+
+    static_web_server_host: AnyHttpUrl = Field(
```

Comment:
This is rather an ``url`` than a ``host``. Suggest ``static_web_server_url``

Reply:
yes, good one!